command/project-create.go

```diff
@@ -116,6 +133,21 @@ func (c *ProjectCreateCommand) Run(args []string) int {
 		payload["slack_webhook"] = *slackURL
 	}
 
+	var integrated_services []map[string]string
+
+	if *consulEnabled {
+		integrated_services = append(integrated_services, map[string]string{"name": "consul", "enabled": "true", "basicauth": *consulBasicAuth, "prefix": *consulPrefix, "ipwhitelist": *consulIpWhiteList})
```

Comment:
I am not sure anymore on how empty strings are treated on the web side as opposed to no corresponding section but if I remember the UI/front part this should be ok but still to be verified